apps/finance/app/src/components/Balances.js

```diff
@@ -114,12 +125,22 @@ class Balances extends React.Component {
               `}
             >
               {balanceItems.map(
-                ({ address, amount, convertedAmount, symbol, verified }) => (
+                ({
+                  address,
+                  amount,
+                  convertedAmount,
+                  decimals,
+                  symbol,
+                  verified,
+                }) => (
                   <li
                     key={address}
                     css={`
+                      flex-shrink: 0;
                       display: block;
                       min-width: ${20 * GU}px;
+                      padding-right: ${3 * GU}px;
```

Comment:
```suggestion
```

😄

Reply:
this padding was not necessary ? 🐙

Reply:
I messed up during the merge, there is another `padding-right` [right below](https://github.com/aragon/aragon-apps/pull/1121/files/1dfd55665a673af8007b3365b1dbeff62f87df0b#diff-e842cf2b89a742592b21eeebff9f7086R143) :see_no_evil: